index.html

```diff
@@ -42,19 +46,21 @@
   </head>
   <body>
     <header>
-      <nav>
-        <div class="logo-heading">
-          <a href="#" class="my-logo">My Logo</a>
-        </div>
-        <a title="menu" href="#" class="hamburger-menu">
-          <i class="fa fa-bars"></i>
-        </a>
-        <div class="nav-menu">
-          <a title="portfolio" href="#" class="nav-menu-link">Portfolio</a>
-          <a title="about" href="#" class="nav-menu-link">About</a>
-          <a title="contact" href="#" class="nav-menu-link">Contact</a>
-        </div>
-      </nav>
+      <div class="fixed">
+        <nav>
+          <div class="logo-heading">
+            <a href="#" class="my-logo">My Logo</a>
+          </div>
+          <a title="menu" href="#" class="hamburger-menu">
+            <i class="fa fa-bars"></i>
+          </a>
+          <div class="nav-menu">
+            <a title="portfolio" href="#works" class="nav-menu-link">Portfolio</a>
+            <a title="about" href="#about-me" class="nav-menu-link">About</a>
+            <a title="contact" href="#contact-with-me" class="nav-menu-link">Contact</a>
+          </div>
+        </nav>
+      </div>
```

Comment:
- [ ]  Good job implementing this required change. There's just a little tweak necessary for you to make. The fixed navbar clashes with the content on the page. The background of the navbar is slightly see-through. Kindly fix this to increase accessibility and overall page aesthetics. 
- [ ] Also, consider making the navbar full width of the screen, on the desktop.
_See image below_

![image](https://user-images.githubusercontent.com/49621279/122203162-9822f500-ce52-11eb-97ed-4a5fca98e47d.png)

Reply:
Thank you for your feedbacks. I fixed them all.